### PR TITLE
feat: init mobile entry

### PR DESCRIPTION
**Pull request by pengx17**

### CODE REVIEW

**Comment by pengx17** [comment]
* **#7905** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7905?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/workbench/view/mobile/workbench-root.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useLiveData, useService } from '@toeverything/infra';
+import { useEffect } from 'react';
+import { type RouteObject, useLocation } from 'react-router-dom';
+
+import { WorkbenchService } from '../../services/workbench';
+import { useBindWorkbenchToBrowserRouter } from '../browser-adapter';
+import { ViewIslandRegistryProvider } from '../view-islands';
+import { ViewRoot } from '../view-root';
+
+export const MobileWorkbenchRoot = ({ routes }: { routes: RouteObject[] }) => {
```
Comment (EYHN) [inline packages/frontend/core/src/modules/workbench/view/mobile/workbench-root.tsx]: ~~What if we just use desktop version WorkbenchRoot?~~

Comment (EYHN) [reply packages/frontend/core/src/modules/workbench/view/mobile/workbench-root.tsx]: move this file to `packages/frontend/mobile/src/modules/workbench/view/workbench-root.tsx` ?

**Review by EYHN — APPROVED** [review]

File: packages/frontend/core/src/modules/workbench/view/mobile/workbench-root.tsx (hunk 1 above)
Comment (EYHN) [reply packages/frontend/core/src/modules/workbench/view/mobile/workbench-root.tsx]: or `packages/frontend/mobile/src/pages/workspace/workbench-root.tsx`

Comment (EYHN) [reply packages/frontend/core/src/modules/workbench/view/mobile/workbench-root.tsx]: This is mobile-only, i think we should put it in @affine/mobile

**Pull request merged**